cardano-cli/test/cardano-cli-golden/Test/Golden/Latest/Transaction/CalculateMinFee.hs

```diff
@@ -62,18 +93,18 @@ hprop_golden_shelley_transaction_calculate_min_fee = do
 
       case flags of
         [] ->
-          Aeson.decode (TL.encodeUtf8 (TL.pack minFeeTxt)) === Just (Aeson.object ["fee" .= (2050100 :: Int)])
+          Aeson.decode (TL.encodeUtf8 (TL.pack minFeeTxt)) === Just (Aeson.object ["fee" .= (247473 :: Int)])
         ["--output-text"] ->
-          H.diff minFeeTxt (==) "2050100 Lovelace\n"
+          H.diff minFeeTxt (==) "247473 Lovelace\n"
         ["--output-text", "--out-file"] -> do
           textOnDisk <- H.readFile outFileFp
-          H.diff textOnDisk (==) "2050100 Lovelace"
+          H.diff textOnDisk (==) "247473 Lovelace"
         ["--output-json"] -> do
           let jsonFromStdout = Aeson.decode $ TL.encodeUtf8 $ TL.pack minFeeTxt
-          H.diff jsonFromStdout (==) (Just $ Aeson.object ["fee" .= (2050100 :: Int)])
+          H.diff jsonFromStdout (==) (Just $ Aeson.object ["fee" .= (247473 :: Int)])
         ["--output-json", "--out-file"] -> do
           jsonOnDisk :: Aeson.Value <- H.readJsonFileOk outFileFp
-          H.diff jsonOnDisk (==) (Aeson.object ["fee" .= (2050100 :: Int)])
+          H.diff jsonOnDisk (==) (Aeson.object ["fee" .= (247473 :: Int)])
```

Comment:
Why the fee has changed?

Reply:
I am using a Conway era transaction now. Previously it was Babbage,